zigbee/Zigbee-Preparatory-Course.md

```diff
@@ -165,6 +186,8 @@ For Silicon Labs customer, you can get a 30-day evaluation license for IAR:
 As the [IoT-Developer-Boot-Camp](https://github.com/MarkDing/IoT-Developer-Boot-Camp) is a open source project, we use the github to maintain all of the source code and documentations, every participating in it can provide your comments on it, submit your pull request for bug fix, new feature implementation or documentation improvement.  
 A valid github account is the prerequisite for joining this project, please [Create your github account](https://github.com/join?source=header-home) if you do not have that. And then Fork the [IoT-Developer-Boot-Camp](https://github.com/MarkDing/IoT-Developer-Boot-Camp) repository and add it to your watching list.  
```

Comment:
perhaps we should suggest reader to form the IoT-Developer-Boot-Camp-Wiki repo. What do you think?